packages/transformers/svg-react/src/SvgReactTransformer.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+
+import path from 'path';
+import camelcase from 'camelcase';
+import svgToJsx from 'svg-to-jsx';
+
+import * as babelParser from '@babel/parser';
+
+function getComponentName(filePath) {
+  let validCharacters = /[^a-zA-Z0-9_-]/g;
+  let name = path.parse(filePath).name.replace(validCharacters, '');
+  let pascalCaseFileName = camelcase(name, {
+    pascalCase: true,
+  });
+
+  return `Svg${pascalCaseFileName}`;
+}
+
+export default new Transformer({
+  async parse({asset}) {
+    let code = await asset.getCode();
```

Comment:
If you move the logic from parse to transform you don't have to return an ast and can just use asset.setcode and return the asset wihtout needing babel at all.

That's what we've done in a couple transformers that dont' utilise ast's